src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -406,6 +407,11 @@ private String clientSetname(final String name) {
     return getStatusCodeReply();
   }
 
+  /**
+   * @deprecated The quit command is deprecated and disconnect is recommended, see
+   * <a href="https://github.com/redis/redis/issues/11420">#11420</a>
```

Comment:
```suggestion
   * @deprecated The QUIT command is deprecated, see <a href="https://github.com/redis/redis/issues/11420">#11420</a>.
   * {@link Connection#disconnect()} can be used instead.
```

Reply:
@sazzad16 How about recommending users to use 'Jedis. close()'? (Its also to call disconnect())

Reply:
@yangbodong22011 If the object (Connection/Jedis) is received from a Pool, close() would just return it to the pool, not disconnect.